orderer/consensus/etcdraft/consenter.go

```diff
@@ -160,10 +162,12 @@ func (c *Consenter) HandleChain(support consensus.ConsenterSupport, metadata *co
 	id, err := c.detectSelfID(consenters)
 	if err != nil {
 		if c.InactiveChainRegistry != nil {
-			// There is a system channel, use the InactiveChainRegistry to track the future config updates of application channel.
+			// There is a system channel, use the InactiveChainRegistry to track the
+			// future config updates of application channel.
 			c.InactiveChainRegistry.TrackChain(support.ChannelID(), support.Block(0), func() {
 				c.ChainManager.CreateChain(support.ChannelID())
 			})
+			c.ChainManager.ReportClusterRelationAndStatusMetrics(support.ChannelID(), types.ClusterRelationConfigTracker, types.StatusInactive)
```

Comment:
I _really_ do not like this.  And this is one of the strong reasons why I asked that we please ditch the notion of the raft consenter switching the chain out, and delegate this back up to the multichannel registrar.  Consensus plugins have no business setting cluster relation status.

Reply:
Yes, I did not like doing this but it's the only way I found with the current code flow.

Reply:
Do we have an outstanding JIRA to fix this flow?

Reply:
No, I do not see a JIRA for this. I can try to create one but I'm not sure if I have enough context in this area to get it completely right.

Reply:
There is a Jira for it, or a variation on this theme: https://jira.hyperledger.org/browse/FAB-18152